source/mqtt.c

```diff
@@ -143,7 +143,13 @@ bool aws_mqtt_is_valid_topic_filter(const struct aws_byte_cursor *topic_filter)
                 "Resubscribe not necessary, client is not subscribed to any topics."),
             AWS_DEFINE_ERROR_INFO_MQTT(
                 AWS_ERROR_MQTT_CONNECTION_SHUTDOWN,
-            "Mqtt operation interrupted by connection shutdown"),
+                "Mqtt operation interrupted by connection shutdown."),
+            AWS_DEFINE_ERROR_INFO_MQTT(
+                AWS_ERROR_MQTT_CONNECTION_DESTORIED,
+                "Destory connection has been called, all uncompleted requests will fail."),
```

Comment:
Destroy/DESTROYED

Reply:
I want to mean `aws_mqtt_client_connection_destroy` is called...